src/client/store/actions/postUserDataAction.test.js

```diff
@@ -0,0 +1,72 @@
+import configureMockStore from 'redux-mock-store';
+import 'babel-polyfill';
+import thunk from 'redux-thunk';
+import axios from 'axios';
+
+import {
+  AUTH_START,
+  POST_USER_DATA,
+  AUTH_FAIL,
+} from './types';
+
+import {
+  postUserData,
+} from './postUserDataAction';
+
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+
+describe('TEST POST ACTIONS', () => {
+  describe('ASYNC POST ACTION', () => {
+    beforeEach(() => {
+      jest.resetAllMocks();
+      jest.restoreAllMocks();
```

Comment:
Are this two methods required, or could be used just one of them?

Reply:
)